modules/oe_theme_content_news/oe_theme_content_news.module

```diff
@@ -7,25 +7,39 @@
 
 declare(strict_types = 1);
 
+use Drupal\Core\Cache\CacheableMetadata;
 use Drupal\media\MediaInterface;
+use Drupal\rdf_skos\Entity\ConceptInterface;
 
 /**
  * Implements template_preprocess_node() for the News node type.
  */
 function oe_theme_content_news_preprocess_node__oe_news(&$variables) {
-  if ($variables['view_mode'] !== 'full') {
-    return;
-  }
-
   /** @var \Drupal\node\NodeInterface $node */
   $node = $variables['node'];
 
-  $media = $node->get('oe_news_featured_media')->entity;
-  if ($media instanceof MediaInterface) {
-    $builder = \Drupal::entityTypeManager()->getViewBuilder('media');
-    $variables['content']['featured_image'] = $builder->view($media, 'oe_theme_main_content');
-    // Show the image always at the top.
-    $variables['content']['featured_image']['#weight'] = -100;
-    $variables['content']['featured_image']['#attributes']['class'][] = 'ecl-u-type-paragraph';
+  if ($variables['view_mode'] === 'full') {
+    $media = $node->get('oe_news_featured_media')->entity;
+    if ($media instanceof MediaInterface) {
+      $builder = \Drupal::entityTypeManager()->getViewBuilder('media');
+      $variables['content']['featured_image'] = $builder->view($media, 'oe_theme_main_content');
+      // Show the image always at the top.
+      $variables['content']['featured_image']['#weight'] = -100;
+      $variables['content']['featured_image']['#attributes']['class'][] = 'ecl-u-type-paragraph';
+    }
+  }
+
+  if ($variables['view_mode'] !== 'teaser') {
+    return;
+  }
+  $news_type = $node->get('oe_news_types')->entity;
+  if ($news_type instanceof ConceptInterface) {
+    $cacheability = new CacheableMetadata();
+    $cacheability->addCacheableDependency($news_type);
+    $news_type = \Drupal::service('entity.repository')
+      ->getTranslationFromContext($news_type);
+    $variables['content']['news_types'] = $news_type->label();
   }
+  $timestamp = $node->get('oe_publication_date')->date->getTimestamp();
+  $variables['content']['date'] = \Drupal::service('date.formatter')->format($timestamp, 'oe_theme_news_date');
```

Comment:
Restore this as it was before, we can simply use drupal fields now, we export the teaser view mode for this reason.